src/daemon/main.rs

```diff
@@ -157,18 +158,41 @@ impl Daemon {
     }
 }
 
-fn determine_client_spacial_attributes(
+fn determine_client_spatial_attributes(
     index: i32,
     number_of_consoles: i32,
     workspace_area: &workspace::WorkspaceArea,
 ) -> (i32, i32, i32, i32) {
-    // FIXME: if number of hosts is < 3 or =5 determination of client spacial attributes falls apart
-    let height_width_ratio = workspace_area.height as f64 / workspace_area.width as f64;
-    let number_of_columns = (number_of_consoles as f64 / height_width_ratio).sqrt() as i32;
-    let console_width = workspace_area.width / number_of_columns;
-    let console_height = (console_width as f64 * height_width_ratio) as i32;
-    let x = workspace_area.width / number_of_columns * (index % number_of_columns);
-    let y = index / number_of_columns * console_height;
+    let aspect_ratio = workspace_area.width as f64 / workspace_area.height as f64;
+
+    const ASPECT_RATIO_ADJUSTMENT: f64 = 0.5;
```

Comment:
How about moving this to the top of the file below the KEY_EVENT constant?
I would also suggest `0f64` as default value, what do you think?

Reply:
Done